pkg/skaffold/docker/image.go

```diff
@@ -364,6 +365,10 @@ func (l *localDaemon) Build(ctx context.Context, out io.Writer, workspace string
 	}
 
 	if err := streamDockerMessages(out, resp.Body, auxCallback); err != nil {
+		var jm *jsonmessage.JSONError
+		if errors.As(err, &jm) {
+			return "", err
```

Comment:
Maybe this should be 
```suggestion
			return "", fmt.Errorf("docker build failure: %w", err)
```

Reply:
I like it better too.